lib/i18n/hygiene/config.rb

```diff
@@ -0,0 +1,7 @@
+module I18n
+  module Hygiene
+    class Config
+      attr_accessor :locales
```

Comment:
Some of the checks might one day need a "main locale" config option. Does `locales` signal what the difference between those two options would be? I'm struggling to come up with anything better though...